cdci_data_analysis/analysis/parameters.py

```diff
@@ -447,9 +451,12 @@ def __init__(self, value=None, T_format='isot', name=None, Time_format_name=None
                          default_units='isot',
                          name=name)
 
-    def get_value_in_default_format(self) -> Union[str, float, None]:
+    def get_value_in_default_units(self) -> Union[str, float, None]:
         return getattr(self._astropy_time, self.default_units)
 
```

Comment:
```suggestion

    def get_value_in_default_format(self):
        return self.get_value_in_default_units()

```

Reply:
name of the parameter, "Time" is a bit confusing. It implies TimeStamp, a moment of time.Such an entity has no units, but has a format. Only things that inherit from Float can have units. Methods could be arranged accordingly: unit-specific methods can be moved from `Parameter` to `Float`.

Reply:
And having the concepts of units/type/format removed from the class Parameter ?